### DISCUSSION TITLE
“攻读学位期间获得的科研成果”，如何专利和参与项目分开写。

### DISCUSSION THREAD

**Discussion by GodfatherPacino** [sjtug/SJTUThesis#787] [提问]
您好：
我想在“攻读学位期间获得的科研成果”中，写上“专利”和“参与项目”两部分内容。但是目前的模板，只能通过`
\begin{achievements}
  \item xxx
\end{achievements}`
的方式去加。只能加一种。
请问有什么好的解决方法。

**Comment by LogCreative** [comment]
或许现在示例文档中的 [contents/achievements.tex](https://github.com/sjtug/SJTUThesis/blob/master/contents/achievements.tex) 的内容就是你想要的格式？

**Reply by GodfatherPacino** [reply]
我用的是上一个版本，1.0.0，请问在那个版本中有办法修改吗？（因为我在这个版本中已经对cls文件和其他一些文件作出了修改，怕兼容性问题）。

**Reply by skyzh** [reply]
旧版本已经不维护了。
